web/thesauruses/c/17/functions.json

```diff
@@ -0,0 +1,72 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "functions"
+    },
+    "concepts": {
+        "void_functions_no_parameters": {
```

Comment:
Same here. You can't delete or change the IDs, they're actually needed.

![image](https://user-images.githubusercontent.com/2601974/197648356-7a391c9c-49e8-4d72-af7a-9ad499550701.png)